server/README.md

```diff
@@ -249,3 +249,34 @@ cargo run --bin remote_server_cli -- initialise-from-central -u 'user1:password1
 # attempt to refresh dates (advance them forward, see --help)
 cargo run --bin remote_server_cli -- refresh-dates
 ```
+
+
+# Making Mac Demo Binary with Data
+
+```bash
+# Buid and copy
+cd ../client && yarn install && yarn build
+cd ../server && cargo clean && cargo build --release --bin remote_server
+
+rm -rf m1_mac_sqlite && mkdir m1_mac_sqlite && mkdir m1_mac_sqlite/bin && mv target/release/remote_server m1_mac_sqlite/bin 
+# Copy configurations
+mkdir m1_mac_sqlite/configuration && cp configuration/base.yaml m1_mac_sqlite/configuration/
+touch  m1_mac_sqlite/configuration/local.yaml
+
+# Create script to start server
+echo '
+#!/bin/bash
+cd "$(dirname "$0")"
+chmod +x bin/remote_server
+xattr -cr bin/remote_server
```

Comment:
Mac automatically adds certain attributes to executables that make them look like they are corrupt (for security), this removes it